doc/guide/packages.xml

```diff
@@ -347,17 +347,13 @@ $ cockpit-bridge --packages
       more than one extension, the second to the last extension is popped off, and the above
       process repeats.</para>
 
-    <para>This means that for the file <code>test.de.js</code> in the package named
+    <para>This means that for the file <code>test.js</code> in the package named
       <code>mypackage</code> the following files would be tried in this order:</para>
```

Comment:
Unfortunately this is not true and it's not how the implementation works.

.js.gz and .js are treated equivalently.  The one that gets served will be whichever one was found last.  .min.js.gz and .min.js are also treated equivalently, but due to an implementation quirk, the one that is found first will win.  Not that the order of the readdir() syscall is important to the user...

I'd maybe split this into two parts and basically say that .gz and non-.gz files are treated more or less equivalently and then note that requesting test.js will fall back to test.min.js if test.js isn't found.

You could also take a more direct tack and describe that we scan all the files, dropping .gz suffixes and also serving .min. files as their base versions as a fallback.  This might be harder to understand, though.

Reply:
Right, thanks! I reworked the whole paragraph after checking the source again, there were some other obsolete things.